examples/correlation_matrices/sampler.cpp

```diff
@@ -147,6 +150,61 @@ void correlation_matrix_uniform_sampling_MT(const unsigned int n, const unsigned
     write_to_file<PointMT>(walkname + "_matrices_MT" + std::to_string(n) + ".txt", randPoints);
 }
 
+template<typename WalkTypePolicy, typename PointType, typename RNGType, typename MT>
+void tune_parameter_L(const int walkL, const int choice, const std::vector<unsigned int>& dimensions, const unsigned int num_points,
+            	      const unsigned int nburns, const unsigned int num_matrices){
```

Comment:
What is `num_matrices`? Is this something different than the `num_points`?

Reply:
I was testing this function from main() and I had run this function in a loop that iterates num_matrices number of times to look at the variation in run times; but this loop isn't required for the function, I have removed it.